packages/react/src/MinimalFooter/MinimalFooter.tsx

```diff
@@ -212,6 +212,14 @@ function SocialLogomarks({socialLinks, logoHref}: SocialLogomarksProps) {
       icon: 'https://github.githubassets.com/images/modules/site/icons/footer/tiktok.svg',
       iconWidth: 18,
       iconHeight: 18
+    },
+    {
+      name: 'instagram',
```

Comment:
Could you also update the doc prop table please to reference Instagram.

![Screenshot 2023-05-30 at 09 30 56](https://github.com/primer/brand/assets/13340707/e2a335bb-1c29-4623-8953-cf52a3275b42)

Reply:
This has been resolved in the following [SHA](https://github.com/primer/brand/pull/282/commits/916b5dbe64cb99e76ae8d4527aaeaf09f6434f9b)

Reply:
This has been resolved in the following [SHA](https://github.com/primer/brand/pull/282/commits/3c23a310137a9d2b481c10fb5b2400eb4f23feeb)